src/components/List/ListCard/ListCardIcons/ListCardIcons.tsx

```diff
@@ -0,0 +1,19 @@
+import { AiOutlinePaperClip } from "react-icons/ai";
+import { GrTextAlignFull } from "react-icons/gr";
+import { ListCardIconProps } from "../../../../types/components/ListCardIconProps";
+
+const ListCardIcons = ({ card }: ListCardIconProps) => {
+  return (
+    <div className="flex items-center space-x-2 text-gray-600 pt-2">
+      {card.description.length > 0 && <GrTextAlignFull />}
+      {card.attachments.length > 0 && (
+        <div className="flex justify-center">
+          <AiOutlinePaperClip size={20} />
```

Comment:
I've slightly tweaked the attachments icon to be bigger as well as the attachments text as the description icon was bigger and it looked odd i.e.

With the old classes:
![image](https://user-images.githubusercontent.com/43914992/175375612-05aafb58-db6f-4c2f-adce-5e42d848cc84.png)

With the new classes:
![image](https://user-images.githubusercontent.com/43914992/175375782-f99adfe5-3ad5-4071-8fde-7a425ca6d256.png)